sql/2021/pwa/sw_objects.sql

```diff
@@ -0,0 +1,51 @@
+#standardSQL
+# SW objects
+CREATE TEMPORARY FUNCTION getSWObjects(swObjectsInfo STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+try {
+  var swObjects = Object.values(JSON.parse(swObjectsInfo));
+  if (typeof swObjects != 'string') {
+    swObjects = swObjects.toString();
+  }
+  swObjects = swObjects.trim().split(',');
+  return Array.from(new Set(swObjects));
+} catch (e) {
+  return [e];
+}
+''';
+SELECT
+  _TABLE_SUFFIX AS client,
+  sw_object,
+  COUNT(DISTINCT url) AS freq,
+  total,
+  COUNT(DISTINCT url) / total AS pct
+FROM
+  `httparchive.sample_data.pages_*`,
+  --`httparchive.pages.2021_07_01_*`,
```

Comment:
Does this need to be rerun?

```suggestion
  `httparchive.pages.2021_07_01_*`,
```

Reply:
Reran and got same numbers as sheet so must have just checked in the wrong code.